src/fields/Fields.cpp

```diff
@@ -981,32 +1079,36 @@ Fields::MixAndShiftBfields (const amrex::MultiFab& B_iter, amrex::MultiFab& B_pr
         weight_B_prev_iter = 0.5_rt;
     }
 
+    amrex::MultiFab& slicemf = getSlices(lev);
+
+    AMREX_ALWAYS_ASSERT(Comps[WhichSlice::This]["Bx"]+1==Comps[WhichSlice::This]["By"]);
+    AMREX_ALWAYS_ASSERT(Comps[WhichSlice::PCIter]["Bx"]+1==Comps[WhichSlice::PCIter]["By"]);
+    AMREX_ALWAYS_ASSERT(Comps[WhichSlice::PCPrevIter]["Bx"]+1==Comps[WhichSlice::PCPrevIter]["By"]);
+
     /* calculating the mixed temporary B field  B_prev_iter = c*B_iter + d*B_prev_iter.
      * This is temporarily stored in B_prev_iter just to avoid additional memory allocation.
      * B_prev_iter is overwritten at the end of this function */
```

Comment:
```suggestion
    /* calculating the mixed temporary B field  B[WhichSlice::PCPrevIter] = c*B[WhichSlice::PCIter] + d*B[WhichSlice::PCPrevIter].
     * This is temporarily stored in B[WhichSlice::PCPrevIter] just to avoid additional memory allocation.
     * B[WhichSlice::PCPrevIter] is overwritten at the end of this function */
```
For consistency with the code